### DISCUSSION TITLE
MVT Layer loadoptions update trigger not updating

### DISCUSSION THREAD

**Discussion by dkspec** [visgl/deck.gl#6172] [Q&A]
Hi,
I have a MVT Layer with accesstoken that refreshes every 30mins.
When token refreshes, it doesn't seem to pick up the new value when using updatetrigger. Is this supported?
Is there another way I should go about?

new MVTLayer({
    id: 'mvt-layer-accesstoken',
    data: `my/api/...`,
    loadOptions: {
        headers: {
            Authorization: 'Bearer ' + authToken,
        },
    },
    updateTriggers: {
        loadOptions: authToken
    },
})

**Comment by Pessimistress** [comment]
`updateTriggers` is only for accessors, see https://deck.gl/docs/api-reference/core/layer#updatetriggers

Rerendering the layer with a new `loadOptions` value should suffice. If it doesn't work, please open a bug.

**Comment by sprezz13** [comment]
@dkspec did rerendering fix your issue? I am having the same issue with the MVT Layer as well

**Comment by dkspec** [comment]
@sprezz13 i triggered the rerender by chaning the url of data layer with version number
I am hoping to avoid this as it reloads the data when accessToken is changed.
